.jshintrc

```diff
@@ -8,6 +8,6 @@
 	"nonbsp": true,
 	"undef": true,
 	"unused": true,
-	"strict": "global",
+	"strict": "implied",
 	"varstmt": true
 }
```

Comment:
Also `.jshintrc` file from `test/` directory should be changed since we write tests using ES6 and after this task execution `'use strict';` is removed from test files. Additionally, this PR should remove `'use strict';` from test files located in `ckeditor5` repository.